pydy/system.py

```diff
@@ -545,7 +578,15 @@ def _Kane_constant_symbols(self):
         TODO temporary.
 
         """
-        constants = list(self.eom_method._find_othersymbols(
-            *self._Kane_inlist_insyms()))
-        constants.remove(sym.symbols('t'))
+        from_eoms, from_sym_lists = self._Kane_inlist_insyms()
+        if cmp(parse_version(SYMPY_VERSION), parse_version('0.7.6')) > -1:
```

Comment:
This is repeated frequently throughout this patch. Perhaps this could be turned into a function? Or better yet, evaluated once, and stored in a global somewhere?